src/main/kotlin/org/rust/ide/intentions/createFromUsage/CreateFunctionIntention.kt

```diff
@@ -267,8 +253,10 @@ class CreateFunctionIntention : RsElementBaseIntentionAction<CreateFunctionInten
             }
         }
 
-        val newImpl = RsPsiFactory(item.project).createInherentImplItem(item.name
-            ?: "?", item.typeParameterList, item.whereClause)
+        val newImpl = RsPsiFactory(item.project).createInherentImplItem(
+            item.name
+                ?: "?", item.typeParameterList, item.whereClause
+        )
```

Comment:
This is quite a weird formatting. Is it a work of our formatter? =D 
Let's just put parameters on separate lines and leave `item.name ?: "?"` on one line.